pkgs/development/python-modules/kaleido/tests.nix

```
@@ -0,0 +1,15 @@
+{
+  runCommand,
+  python3,
```

```suggestion
  python,
```

Missed this earlier, oops. (We really should have a check for this.)